src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -709,10 +709,12 @@ def handle_timeline_added_event(
         except Exception:
             individual = None
 
+        source = f"Slack message from {individual.name}" if individual else "Slack message"
```

Comment:
Any chance we could include the name of the App/bot?

Reply:
Good idea - done!